ui/k-info-panel/edit_window.kytos

```diff
@@ -634,6 +641,48 @@
                     _this.$kytos.$emit("setNotification", notification)
                 })
             },
+            /*
+            Extends a maintenance window.
+            */
+            extendWindow() {
+                _this = this
+
+                // Call the Maintenace API to extend the maintenance window.
+                var request = $.ajax({
+                    url: this.$kytos_server_api + "kytos/maintenance/" + _this.window_data.Id + "/extend",
+                    type: "PATCH",
+                    dataType: "json",
+                    contentType: "application/json",
+                    data: JSON.stringify({"minutes": _this.minutes,
```

Comment:
`_this.minutes` will bind as a string, this will result in a type error since the API expects an integer, can you parse and cast the number as an integer?